### PR TITLE
MS-655 task: reuse redis Object

### PR DESCRIPTION
**Pull request by jjrobothamho**
Reuse redis Object for efficient data store.

### CODE REVIEW

File: test/unit/nrm/models/check-token.test.js [hunk 1]
```diff
@@ -1,59 +1,68 @@
...
-      Model.read('test')
-        .then((result) => {
-            result.should.to.deep.equal(expected);
-            done();
-          })
-          .catch(err=> console.log(err));
+
+      try {
+        const result = await Model.read('test');
+        console.log(result);
```
Comment (sulthan-ahmed) [inline test/unit/nrm/models/check-token.test.js]: did you mean to `console.log` it?

File: redis.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+const Redis = require('ioredis');
+const config = require('./config');
+
+const redis = new Redis({
+  port: config.redis.port,
+  host: config.redis.host,
+  connectionName: 'modern-slavery-app',
+});
+
+module.exports = redis;
```
Comment (sulthan-ahmed) [inline redis.js]: FANTASTIC WORK 🔥

**Comment by sulthan-ahmed** [comment]
Super dooper work, just some minor changes

**Comment by sulthan-ahmed** [comment]
updating branch

File: test/unit/nrm/models/check-token.test.js [hunk 3]
```diff
@@ -1,59 +1,68 @@
...
-  {'ioredis': redis});
+{
+  '../../../redis': redis,
+});
 
 describe('apps/nrm/models/check-token', () => {
   describe('read()', ()=> {
     beforeEach(() => {
-      sinon.stub(redis.prototype, 'get');
+      sinon.stub(redis, 'get');
```
[the remark this answers is not in the document]
Comment (jjrobothamho) [reply test/unit/nrm/models/check-token.test.js]: No, the object was empty when I checked via console log. Might need to double check.

[the remark this answers is not in the document]
Comment (jjrobothamho) [reply test/unit/nrm/models/check-token.test.js]: redis.prototype is undefined

**Comment by sulthan-ahmed** [comment]
@jjrobothamho if this is ready to review, remove the WIP

File: test/unit/nrm/models/check-token.test.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (sulthan-ahmed) [reply test/unit/nrm/models/check-token.test.js]: yeah but was it undefined without the Callthru? just curious

**Comment by sulthan-ahmed** [comment]
I removed WIP for you @jjrobothamho 
I'd probably remove WIP when you want something to be merged

**Comment by sulthan-ahmed** [comment]
lgtm

**Pull request merged**